exercise.js

```diff
@@ -0,0 +1,15 @@
+const colors = [
+  "red",
+  9999,
+  "green",
+  "blue",
+  777,
+  "yellow",
+  "black",
+  "purple",
+  "brown",
+];
```

Comment:
The way you solved is correct, But I am going to show you the best solution, so imagine you have hundred or maybe thousands of elements in an Array so how you can switch all numbers to string! Here is the solution!
const colors = ["red", 9999, "green", "blue",777, "yellow", "black", "purple", "brown"];
let colorPicker = Math.floor(Math.random() * colors.length);
let filtered = typeof colors[colorPicker] === "number" ? "" : colors[colorPicker];
console.log(filtered);